### PR TITLE
V5 ga 7 tooltip

### PR DESCRIPTION
**Pull request by SrdjanReblaze**
https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-7

### CODE REVIEW

File: src/components/messages/ToolTip.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="tooltip-holder">
+    <span
+      v-if="isActive"
+      class="tooltip"
+      :class="withFadeInAnimation ? 'fade-in' : ''"
+    >
```
Comment (Aviv-Galmidi) [inline src/components/messages/ToolTip.vue:7]: Please keep the closing tag in the same line as the last attribute

File: src/components/messages/ToolTip.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="tooltip-holder">
+    <span
+      v-if="isActive"
```
Comment (Aviv-Galmidi) [inline src/components/messages/ToolTip.vue:4]: Please keep the first attribute line in the same line as the HTML element

File: src/components/messages/ToolTip.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+</script>
+
+<style lang="scss">
+@import 'src/assets/styles/colors';
+
+.tooltip-holder {
+  position: relative;
+
+  .tooltip {
+    background: $datk-blue;
```
Comment (Aviv-Galmidi) [inline src/components/messages/ToolTip.vue:37]: Did you mean `$color-dark-blue`?

File: src/assets/styles/colors.scss [hunk 4]
```diff
@@ -24,6 +24,8 @@ $color-crimson: #dc143c;
 
 $color-curious-blue: #2596d1;
 
+$color-dark-blue: #050038;
```
Comment (Aviv-Galmidi) [inline src/assets/styles/colors.scss:27]: The color name for #050038 is `stratos` not `dark-blue`
https://chir.ag/projects/name-that-color/#050038

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
We should use `Tooltip` rather than `ToolTip` as that's the correct English writing

Please keep the component directory flat `src/components/Tooltip.vue` or something of that fashion rather than src/components/messages/ToolTip.vue

Please add storybook story preview as well

File: src/components/messages/ToolTip.vue (hunk 1 above)
Comment (SrdjanReblaze) [reply src/components/messages/ToolTip.vue]: please, check fresh PR update

**Review by Aviv-Galmidi — COMMENTED** [review]
Approving the code so far, awaiting story before merging

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**